src/test-util/accessibility.js

```diff
@@ -0,0 +1,100 @@
+import { run } from 'axe-core';
+import { ReactWrapper, mount } from 'enzyme';
+import { isValidElement } from 'preact';
+
+/**
+ * @typedef ScenarioProps
+ * @prop {string} [name] -
+ *   A descriptive name for the scenario. Defaults to "default" if not specified.
+ * @prop {() => import('preact').VNode|ReactWrapper} content -
+ *   A function that returns the rendered output to test or an Enzyme wrapper
+ *   created using Enzyme's `mount` function.
+ * @prop {string} [backgroundColor] -
+ *   Background color onto which to render the element. This can affect the
+ *   result of color contrast tests. Defaults to white.
+ */
+
+/** @typedef {ScenarioProps & Object} Scenario */
```

Comment:
_This is a change from the client_ 
Because this file is now exported in `index.js`, we can't skip the typecheck. It was complaining about a missing iterator due to this line
`for (let { name = 'default', content, ...config } of scenarios) {`

Extending the `Object` type seemed to do the trick, but I'm not sure that is best practice here.